cognite/client/_api/hosted_extractors/mappings.py

```diff
@@ -0,0 +1,272 @@
+            list_cls=MappingList,
+            resource_cls=Mapping,
+            method="GET",
+            chunk_size=chunk_size,
+            limit=limit,
+            headers={"cdf-version": "beta"},
+        )
+
+    def __iter__(self) -> Iterator[Mapping]:
+        """Iterate over mappings
+
+        Fetches mappings as they are iterated over, so you keep a limited number of mappings in memory.
+
+        Returns:
+            Iterator[Mapping]: yields Mapping one by one.
+        """
+        return self()
+
+    @overload
+    def retrieve(self, external_ids: str, ignore_unknown_ids: bool = False) -> Mapping: ...
+
+    @overload
+    def retrieve(self, external_ids: SequenceNotStr[str], ignore_unknown_ids: bool = False) -> MappingList: ...
+
+    def retrieve(
+        self, external_ids: str | SequenceNotStr[str], ignore_unknown_ids: bool = False
+    ) -> Mapping | MappingList:
+        """`Retrieve one or more mappings. <https://developer.cognite.com/api#tag/Mappings/operation/retrieve_mappings>`_
+
+        Args:
+            external_ids (str | SequenceNotStr[str]): The external ID provided by the client. Must be unique for the resource type.
+            ignore_unknown_ids (bool): Ignore external IDs that are not found
+
+
+        Returns:
+            Mapping | MappingList: Requested mappings
+
+        Examples:
+
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.hosted_extractors.mappings.retrieve('myMapping')
+
+            Get multiple mapping by id:
+
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> res = client.hosted_extractors.mappings.retrieve(["myMapping", "myMapping2"], ignore_unknown_ids=True)
+
+        """
+        self._warning.warn()
+        return self._retrieve_multiple(
+            list_cls=MappingList,
+            resource_cls=Mapping,
+            identifiers=IdentifierSequence.load(external_ids=external_ids),
+            ignore_unknown_ids=ignore_unknown_ids,
+            headers={"cdf-version": "beta"},
+        )
+
+    def delete(
+        self, external_ids: str | SequenceNotStr[str], ignore_unknown_ids: bool = False, force: bool = False
+    ) -> None:
+        """`Delete one or more mappings  <https://developer.cognite.com/api#tag/Mappings/operation/delete_mappings>`_
+
+        Args:
+            external_ids (str | SequenceNotStr[str]): The external ID provided by the client. Must be unique for the resource type.
+            ignore_unknown_ids (bool): Ignore external IDs that are not found
+            force (bool): Delete any jobs associated with each item.
+
+        Examples:
+
+            Delete dests by id::
+
+                >>> from cognite.client import CogniteClient
+                >>> client = CogniteClient()
+                >>> client.hosted_extractors.dests.delete(["myDest", "MyDest2"])
```

Comment:
I've commented on this one previously I think 😜 Copy/paste errors: `dests` -> `mappings` (dont forget to update external ids)